api/src/main/java/org/openmrs/module/appointmentscheduling/api/AppointmentService.java

```diff
@@ -13,10 +13,7 @@
  */
 package org.openmrs.module.appointmentscheduling.api;
 
-import org.openmrs.Location;
-import org.openmrs.Patient;
-import org.openmrs.Provider;
-import org.openmrs.Visit;
+import org.openmrs.*;
```

Comment:
We don't use wildcard imports by convention in OpenMRS... usually this is something that your editor does automatically, but that you can disable, for instance in IntelliJ, see "Disable Wildcard Imports" here:

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html